### PR TITLE
Fix docker-machine-driver-kvm2 version

### PR DESCRIPTION
**Pull request by josedonizetti**
Change kvm2 version flag, to match minikube as suggested: https://github.com/kubernetes/minikube/pull/4593#discussion_r297356533

WIP: still waiting on the decision of which version to use for the drivers `minikube version` or `commitID`.
discussion at: https://github.com/kubernetes/minikube/pull/4593#discussion_r297310439

### CODE REVIEW

**Comment by medyagh** [comment]
> Change kvm2 version flag, to match minikube as suggested: [#4593 (comment)](https://github.com/kubernetes/minikube/pull/4593#discussion_r297356533)
> 
> WIP: still waiting on the decision of which version to use for the drivers `minikube version` or `commitID`.
> discussion at: [#4593 (comment)](https://github.com/kubernetes/minikube/pull/4593#discussion_r297310439)

we could have both version and commit id in two separate lines. (like current minikube head)

**Comment by josedonizetti** [comment]
```
docker-machine-driver-kvm2 version
version: v1.2.0
gitCommitID: 86cd3c67976adfb6502715f6002ed794880fdf54-dirty
```

@medyagh wdyt?

File: cmd/drivers/kvm/main.go [hunk 1]
```diff
@@ -27,8 +27,9 @@ import (
 )
 
 func main() {
-	if len(os.Args) > 1 && os.Args[1] == "--version" {
-		fmt.Println(kvm.GetVersion())
+	if len(os.Args) > 1 && os.Args[1] == "version" {
+		fmt.Println("version:", kvm.GetVersion())
+		fmt.Println("gitCommitID:", kvm.GetGitCommitID())
```
Comment (medyagh) [inline cmd/drivers/kvm/main.go]: change "gitCommitID:" to "commit"

**Comment by josedonizetti** [comment]
@medyagh done

**Pull request merged**